FAQ.md

```diff
@@ -2,15 +2,23 @@
 
 ## The module provides false positives:
 
-Some SMTP Servers (Yahoo's servers for example) are only rejecting
+The function of this module, and specifically of the SMTP check, relies
+on the assumption that the mail server declared responsible for an email
+domain will immediately reject any nonexistent address.
+
+Some SMTP servers (Yahoo's servers for example) are only rejecting
 nonexistent emails after the end of `DATA` command has been provided in
 the conversation with the server. This module only goes until the
 `RCPT TO` and says it's valid if it doesn't get rejected there, since
-the `DATA` part of the email is the email body itself. There's not much
-one can do with it, you have to accept false positives in the case of
-yahoo.com and some other providers. I'm not sure if rejecting emails
-after the `DATA` command is a valid behavior based on the SMTP RFC, but
-I wouldn't wonder if not.
+the `DATA` part of the email is the email body itself.
+
+Other SMTP servers accept emails even for nonexistent recipient
+addresses and forward them to a different server which will create a
+bounce message in a second step. This is the case for many email domains
+hosted at Microsoft.
+
```

Comment:
Is this really the case with MS servers? So far I've seen outlook servers (if you meant those) rejecting the invalid emails properly.

Reply:
I have tested various domains where the MX is <domainname>.mail.protection.outlook.com; some of them reject invalid emails properly and some don't. If you want I can send you some examples in private email, you can find my email address [here[(https://wiki.fsfe.org/Supporters/reinhard).